### PR TITLE
Change vcl to std:: (affects vidtk related files)

### PR DESCRIPTION
**Pull request by RustyBlue**
vcl_ methods were replaced with std:: as is necessary to build with vidtk.

### CODE REVIEW

**Comment by mwoehlke-kitware** [comment]
This conflicts with #18; any chance of reviewing that first?

File: Applications/VpView/vpVidtkTrackIO.cxx [hunk 1]
```diff
@@ -13,6 +13,7 @@
 #include <vtkVgScalars.h>
 #include <vtkVgTrack.h>
 #include <vtkVgTrackTypeRegistry.h>
+#include <vtkVgUtil.h>
```
Comment (mwoehlke-kitware) [inline Applications/VpView/vpVidtkTrackIO.cxx]: ???

**Comment by RustyBlue** [comment]
Since building is involved, it will likely be a couple days before I can look at #18, but I will see if I can take a look today.

File: Applications/VpView/vpVidtkTrackIO.cxx (hunk 1 above)
Comment (RustyBlue) [reply Applications/VpView/vpVidtkTrackIO.cxx]: That slipped in - I'll remove.

**Comment by RustyBlue** [comment]
@mwoehlke-kitware Take another (quick) look?

**Pull request merged**